inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -685,20 +686,43 @@ linelist_cleaned$age_group <- age_categories(linelist_cleaned$age_years,
 #                                            "15-29 years",
 #                                            "30-44 years", 
 #                                            "45+ years")))
+```
+
+
+
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+create_vars
+
+This chunk will help you construct new variables from other variables.
```

Comment:
maybe chuck in an overview of different types of vars in this seciton?

Reply:
Do you mean like describing each type?
I think you need to have that information before the template. But open to adding.

Reply:
just a short bullet list, date vars, numeric vars, factors,... wahtever else...